src/main/java/net/mcreator/minecraft/MinecraftImageGenerator.java

```diff
@@ -690,7 +690,7 @@ public static BufferedImage generatePotionIcon(Color color) {
 		 * @return Returns generated image.
 		 */
 		public static BufferedImage generateBiomePreviewPicture(Workspace workspace, Color airColor, Color grassColor,
-				Color waterColor, MItemBlock groundBlock, MItemBlock undergroundBlock, int treesPerChunk, int treeType,
+				Color waterColor, MItemBlock groundBlock, MItemBlock undergroundBlock, int treesPerChunk, boolean treeType,
```

Comment:
Revert changes to this file, and check my comment on the code location where you call this method.